--- launch.sh	2020-09-18 00:10:12.521459519 +0200
+++ launch.sh.p	2020-09-18 00:12:54.921526930 +0200
@@ -76,14 +76,14 @@
 
 	# Production Environment
 	INSTALL_DIR="${SUPPORT_DIR}/.."
-	CPATH="${INSTALL_DIR}/Ghidra/Framework/Utility/lib/Utility.jar"
+	CPATH="${INSTALL_DIR}/Ghidra/Framework/Utility/lib/Utility.jar:${INSTALL_DIR}/Ghidra/Framework/Docking/lib/flatlaf-intellij-themes-0.42-20200904.210752-6.jar:${INSTALL_DIR}/Ghidra/Framework/Docking/lib/flatlaf-0.42-20200904.210752-6.jar"
 	LS_CPATH="${SUPPORT_DIR}/LaunchSupport.jar"
 	DEBUG_LOG4J="${SUPPORT_DIR}/debug.log4j.xml"
 else
 
 	# Development Environment
 	INSTALL_DIR="${SUPPORT_DIR}/../../../.."
-	CPATH="${INSTALL_DIR}/Ghidra/Framework/Utility/bin/main"
+	CPATH="${INSTALL_DIR}/Ghidra/Framework/Utility/bin/main:${INSTALL_DIR}/Ghidra/Framework/Docking/lib/flatlaf-intellij-themes-0.42-20200904.210752-6.jar:${INSTALL_DIR}/Ghidra/Framework/Docking/lib/flatlaf-0.42-20200904.210752-6.jar"
 	LS_CPATH="${INSTALL_DIR}/GhidraBuild/LaunchSupport/bin/main"
 	DEBUG_LOG4J="${INSTALL_DIR}/Ghidra/RuntimeScripts/Common/support/debug.log4j.xml"
 fi
