diff --git a/Ghidra/Framework/Project/src/main/java/ghidra/framework/main/SaveDataDialog.java b/Ghidra/Framework/Project/src/main/java/ghidra/framework/main/SaveDataDialog.java
index da43ed036..876a6e62e 100644
--- a/Ghidra/Framework/Project/src/main/java/ghidra/framework/main/SaveDataDialog.java
+++ b/Ghidra/Framework/Project/src/main/java/ghidra/framework/main/SaveDataDialog.java
@@ -265,7 +265,7 @@ public class SaveDataDialog extends DialogComponentProvider {
 		yesButton.setEnabled(false);
 		for (int i = 0; i < files.size(); i++) {
 			checkboxes[i] = new GCheckBox(files.get(i).getName());
-			checkboxes[i].setBackground(Color.white);
+			checkboxes[i].setBackground(Color.darkGray);
 			saveable[i] = files.get(i).canSave();
 			if (!saveable[i]) {
 				String text = files.get(i).getName() + readOnlyString;
